tests/xtro-sharpie/osx.pending

```diff
@@ -344,6 +344,21 @@
 !unknown-native-enum! NSType bound
 !unknown-native-enum! SLComposeViewControllerResult bound
 !wrong-base-type! MDLVoxelArray expected MDLObject actual NSObject
+!incorrect-protocol-member! NSAccessibility::accessibilityChildrenInNavigationOrder is REQUIRED and should be abstract
+!incorrect-protocol-member! NSAccessibility::accessibilityCustomActions is REQUIRED and should be abstract
+!incorrect-protocol-member! NSAccessibility::accessibilityCustomRotors is REQUIRED and should be abstract
+!incorrect-protocol-member! NSAccessibility::setAccessibilityChildrenInNavigationOrder: is REQUIRED and should be bstract
+!incorrect-protocol-member! NSAccessibility::setAccessibilityCustomActions: is REQUIRED and should be abstract
+!incorrect-protocol-member! NSAccessibility::setAccessibilityCustomRotors: is REQUIRED and should be abstract
+!incorrect-protocol-member! CIImageProcessorInput::surface is REQUIRED and should be abstract
+!incorrect-protocol-member! CIImageProcessorOutput::surface is REQUIRED and should be abstract
+!incorrect-protocol-member! INSpeakable::alternativeSpeakableMatches is REQUIRED and should be abstract
+!incorrect-protocol-member! INSpeakable::vocabularyIdentifier is REQUIRED and should be abstract
+!wrong-enum-size! NSSpellingState managed 4 vs native 8
```

Comment:
It's a breaking change (if not new) but it's also a mistake.
API that use this enum must be verified to make sure they are correct.

Reply:
This was previously used as a convenience enum in our bindings from an untyped header enum:
// Untyped enum (NSAttributedString.h). Only used as a convience enum in our API.

They made it an explicit NSInteger enum in 10.13

Reply:
ok, transform that into a comment in the .pending file